spec/system/users/edit_spec.rb

```diff
@@ -53,6 +53,29 @@ def upload_path(file)
     expect(user.reload.preferences['daily_digest']).to eq(true)
   end
 
+  context 'when user updates name' do
```

Comment:
Ensure that you use VSCode that is set up according to the documentation https://docs.pupilfirst.com/developers/development_setup#code-formatting

The string should be in double quotes according to the recent setup.

Reply:
@kaisersakhi 

The replacements are totally expected

I think, even if you don't wrap string literal into double quotes, it will replace with double quotes.
 
We are sticking to "double quotes", The change was recently introduced by Hari → credit: @bodhish